src/View/Helper/PaginatorHelper.php

```diff
@@ -1169,4 +1170,43 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param int $default Default limit for option selecting. Default is Paginator limit.
+     * @param array $options Options for Select tag attributes like class, id or event
+     *
+     * @return string html output.
+     */
+    public function limitSelect($limits = [], $default = null, $options = [])
+    {
+        $out = $this->Form->create(null, ['type' => 'get']);
+
+        if (empty($limits) || !is_array($limits)) {
```

Comment:
The `!is_array()` check can be avoided it you add `array` typehint to the `$limits` argument.